config/application.rb

```diff
@@ -16,6 +16,7 @@ class Application < Rails::Application
     # Settings in config/environments/* take precedence over those specified here.
     # Application configuration should go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded.
+    config.exceptions_app = self.routes
```

Comment:
This shouldn't be on this PR, it's already merged ;-) https://github.com/openaustralia/publicwhip/pull/1136 could you please remove this commit or open a new PR with just the link change commit?